ui/DesignSystem/Component/Placeholder.svelte

```diff
@@ -2,7 +2,7 @@
   export let style = null;
 </script>
 
-<style>
+<style type="text/css">
```

Comment:
why is this changed here and not elsewhere?

Reply:
Oops, this is a debug leftover. Gonna remove.